### PR TITLE
TorchDynamo: skip convolution fusion when convolution's padding is string

### PR DESCRIPTION
**Pull request by XiaobingSuper**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88864
* __->__ #88794
* #88414
* #88412
* #88403

Currently,  the fusion convolution doesn't support the case when padding is a string, we will support it at the next step.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/overrides.py [hunk 1]
```diff
@@ -486,7 +486,10 @@ def fuse_unary(gm: torch.fx.GraphModule):
                 eval_mode = all(not n.training for n in [computation_node, unary_node])
                 if not eval_mode:
                     continue
-
+                if type(computation_node) in [nn.Conv2d] and isinstance(
+                    computation_node.padding, str
+                ):
```
Comment (jgong5) [inline torch/_inductor/overrides.py:491]: Please add a comment (maybe with TODO) on why we don't support padding in `str` now.

Comment (XiaobingSuper) [reply torch/_inductor/overrides.py:505]: Yes, one comment is added.

**Review by jansel — APPROVED** [review]

**Review by jgong5 — APPROVED** [review]
